src/fourier_filtering.jl

```diff
@@ -90,7 +90,7 @@ function fourier_filter(arr, fct=window_gaussian;  kwargs...)
     return fourier_filter!(copy(arr), fct; kwargs...)
 end
 
-function fourier_filter_by_1D_FT!(arr::TA, wins::AbstractVector; transform_win=false, dims=(1:ndims(arr))) where {N, TA <: AbstractArray{<:Complex, N}}
+function fourier_filter_by_1D_FT!(arr::TA, wins::AbstractVector; transform_win=false, keep_integral=false, dims=(1:ndims(arr))) where {N, TA <: AbstractArray{<:Complex, N}}
```

Comment:
Keep_integral?
Sounds a bit weird. Maybe something regarding normalization.

And, docstrings are missing.

Reply:
Is normalize_kernel better?

Reply:
Yes, since we also work with discrete sums and not integrals here.

Reply:
to be consistent with the other arg, I now used "normalize_win".

Reply:
... also added docstrings, but since these concerns only non-exported functions, I did not add them to the docs.